apps/client/src/features/app-settings/panel/general-panel/StyleEditorModal.tsx

```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+import {
+  Button,
+  Flex,
+  HStack,
```

Comment:
cleanup: we dont use layout components from Chakra and have our own. You can search other interfaces in settings to see how things are composed or I can change if you prefer